.gitmodules

```diff
@@ -28,3 +28,6 @@
 [submodule "aws-common-runtime/aws-c-event-stream"]
 	path = crt/aws-c-event-stream
 	url = https://github.com/awslabs/aws-c-event-stream.git
+[submodule "aws-common-runtime/aws-c-iot"]
+	path = crt/aws-c-iot
+	url = https://github.com/awslabs/aws-c-iot.git
```

Comment:
Will need to update the aws-c-iot commit once the changes from my remote branch in aws-c-iot are merged into main.